autoreject/tests/test_ransac.py

```diff
@@ -50,12 +53,146 @@ def test_ransac():
 
     # should not contain both channel types
     picks = mne.pick_types(epochs.info, meg=True, eeg=False, stim=False,
-                           eog=False, exclude=[])
+                            eog=False, exclude=[])
     ransac = Ransac(picks=picks)
     pytest.raises(ValueError, ransac.fit, epochs)
 
     # should not contain other channel types.
     picks = mne.pick_types(raw.info, meg=False, eeg=True, stim=True,
-                           eog=False, exclude=[])
+                            eog=False, exclude=[])
     ransac = Ransac(picks=picks)
     pytest.raises(ValueError, ransac.fit, epochs)
+
+
+        
+def test_ransac_multiprocessing():
+    """test on real data with 5 random channels augmented with strong
```

Comment:
docstring should be single line. If you want multiline, first do a single line summary and then multi-line afterwards

Reply:
Ah, didn't know, thanks!